doc/workshops/scientific-computing.rst

```diff
@@ -2,4 +2,698 @@
+   :alt: plt.show() help in Spyder's Help Pane
+
+With Seaborn, we can add a line representing the mean for the histogram. To do this we will use the ``axvline()`` method and add a text with the value of the mean with ``text()``. ``min_ylim`` and ``max_ylim`` are used to mark the position of the text.
+
+.. code-block: python
+
+   sns.histplot(data.d_age, kde=True, bins=25)
+   plt.xlabel('Age')
+   plt.axvline(data.d_age.mean(), color='k', linestyle='dashed', linewidth=1)
+   min_ylim, max_ylim = plt.ylim()
+   plt.text(data.d_age.mean()*1.1, max_ylim*0.9,
+            'Mean: {:.2f}'.format(data.d_age.mean()))
+   plt.show()
+
+.. image:: images/workshop-2/age-histogram-2.png
+   :alt: Age histogram with mean line
+
+We know the extent of the dataframe and we have seen how to explore a quantitative variable. Now let's move on to a qualitative variable. The ``columns`` attribute shows a list of column names in the dataframe. We also know that demographic variables are prefixed with "d_", so we can look for variable names that correspond to this prefix.
+
+.. code-block: python
+
+   demograph = [v for v in list(data.columns) if v.startswith("d_")]
+
+In the Variable Explorer you will see a new variable "demograph". If you double click on it you will see a list of the demographic variables in the dataset. At the end of the list you will see a value "d_religion_seriosity". If you right click on it, several options will appear on the screen. You can rename the column, delete it or insert a new one. Let's click on "Copy" to copy the column name into memory.
+
+In the IPython Console, type ``data.`` and then copy the column name. Type a period at the end (no spaces) and hit tab. You should see something like the following:
+
+.. image:: images/workshop-2/console-auto-completion.png
+   :alt: IPython Console auto-completion
+
+You can then see a list of all relevant methods and attributes of the class. Right at the end you will see a "value_counts" that could be used to summarize the variable. It is a method so it must end in parentheses:
+
+.. code-block: python
+
+   data.d_religion_seriosity.value_counts()
+
+   # but not too serious about it     11142
+   # and laughing about it             7307
+   # and somewhat serious about it     5372
+   # and very serious about it         3290
+   # Name: d_religion_seriosity, dtype: int64
+
+In addition to demographic and profile variables, the dataset includes responses to questions on a variety of topics. These questions are coded with a prefix "q" followed by an integer. The text of the questions is stored in a file called "question_data.csv" (which you can download here). [ADD LINK TO DOWNLOAD question_data.csv]
+
+We can examine this file in the Variable Explorer.
+
+.. code-block: python
+
+   question_data = pd.read_csv("question_data.csv", sep=";")
+
+Just double click on "question_data" in the Variable Explorer to display each question aligned with its code ("q" + number).
```

Comment:
In general, using "just" is not advised :) [You can read more about this here](https://software-carpentry.org/blog/2015/03/teaching-tips.html)

```suggestion
Double click on "question_data" in the Variable Explorer to display each question aligned with its code ("q" + number).
```